io.catenax.batch/2.0.0/Batch.ttl

```diff
@@ -0,0 +1,180 @@
+    samm:characteristic :ClassificationCharacteristic;
+    samm:exampleValue "raw material".
+:KeyTrait a samm-c:Trait;
+    samm:preferredName "Key Trait "@en;
+    samm:description "Trait that ensures the usage of predefined keys. "@en;
+    samm-c:baseCharacteristic :KeyCharacteristic;
+    samm-c:constraint ext-batch2:KeyRegularExpression.
+:ValueCharacteristic a samm:Characteristic;
+    samm:preferredName "Value Characteristic"@en;
+    samm:description "The value of an identifier."@en;
+    samm:dataType xsd:string.
+:ProductionCountryCodeTrait a samm-c:Trait;
+    samm:preferredName "Production Country Code Trait"@en;
+    samm:description "Trait to ensure standard data format for country code"@en;
+    samm-c:baseCharacteristic :CountryCodeCharacteristic;
+    samm-c:constraint :CountryCodeRegularExpression.
+:PartIdCharacteristic a samm:Characteristic;
+    samm:preferredName "Part ID Characteristic"@en;
+    samm:description "The part ID is a multi-character string, ususally assigned by an ERP system"@en;
+    samm:dataType xsd:string.
+:PartNameCharacteristic a samm:Characteristic;
+    samm:preferredName "Part Name Characteristic"@en;
+    samm:description "Part Name in string format from the respective system in the value chain"@en;
+    samm:dataType xsd:string.
+:ClassificationCharacteristic a samm-c:Enumeration;
+    samm:preferredName "Classification Characteristic"@en;
+    samm:description "A part type must be placed into one of the following classes: 'component', 'product', 'software', ?assembly?, 'tool', or 'raw material'."@en;
+    samm:dataType xsd:string;
+    samm:see <http://private.pdm-if.org/pdm-if/Frecommended-practices1>;
+    samm-c:values ("product" "raw material" "software" "assembly" "tool" "component").
+:KeyCharacteristic a samm:Characteristic;
+    samm:preferredName "Key Characteristic"@en;
+    samm:description "The key characteristic of a local identifier. A specific subset of keys is predefined. Custom keys are not allowed. Predefined keys:\n- \"manufacturerId\" - The Business Partner Number (BPN) of the manufacturer. Value: BPN-Nummer\n- \"batchId\" - The identifier of the batch, to which the serialzed part belongs"@en;
+    samm:dataType xsd:string.
+ext-batch2:KeyRegularExpression a samm-c:RegularExpressionConstraint;
+    samm:preferredName "Key Regular Expression"@en;
+    samm:description "Constraint that ensures that the predefined keys are used. "@en;
+    samm:value "^(manufacturerId|batchId)$".
+:CountryCodeCharacteristic a samm:Characteristic;
+    samm:preferredName "Country Code Characteristic"@en;
+    samm:description "ISO 3166-1 alpha-3 ? three-letter country codes "@en;
+    samm:dataType xsd:string;
+    samm:see <https://www.iso.org/iso-3166-country-codes.html>.
+:CountryCodeRegularExpression a samm-c:RegularExpressionConstraint;
+    samm:preferredName "Country Code Regular Expression"@en;
+    samm:description "Regular Expression that ensures a three-letter code "@en;
```

Comment:
remove whitespace als please check the remaining document, if there are further places with whitespace and the end of a string (eg L170)

Reply:
done